src/Cameras/babylon.freeCamera.ts

```diff
@@ -114,7 +125,8 @@
 
             globalPosition.subtractFromFloatsToRef(0, this.ellipsoid.y, 0, this._oldPosition);
             this._collider.radius = this.ellipsoid;
-
+	    this._collider._collisionMask = this._collisionMask ? this._collisionMask : -1;

```

Comment:
we should compare to undefined here because mask can be set to 0 and this is valid